src/ether-utils.js

```diff
@@ -0,0 +1,6 @@
+import { isValidAddress } from 'ethereumjs-util';
+
+// Accepts checksummed addresses too
+const EtherAddress = address => isValidAddress(address);
+
+export default EtherAddress;
```

Comment:
We want to add multiple functions to this file, so there is not a function that would obviously be default.